examples/kivy/buildozer.spec

```diff
@@ -225,7 +237,7 @@ android.accept_sdk_license = True
 #android.copy_libs = 1
 
 # (str) The Android arch to build for, choices: armeabi-v7a, arm64-v8a, x86, x86_64
-android.arch = armeabi-v7a
+android.archs = arm64-v8a
```

Comment:
Can we have both Arm arches here?

Or if that doubles the build time, then add a note to the readme that this may need to be changed if targeting older devices.

Reply:
Yes both arm arches are possible, aldough this significantly increases the build time. I have added a note to the readme.